Makefile

```diff
@@ -9,9 +9,14 @@ dependencies:
 	bundle install --verbose
 
 # Run all tests and checks (including linters).
-check:
+check: install  # Installation required for testing binary
 	bundle exec rake
+	sh .buildscript/e2e.sh

```

Comment:
It may be to nice run this e2e test only on CI, since it'll fail locally if you're using a non-Linux machine.

Reply:
Is it okay to rely on the `RUN_E2E_TESTS` env var not being set? The script will terminate early if so.